tools/cli/src/mentat_cli/lib.rs

```diff
@@ -42,6 +42,8 @@ pub fn run() -> i32 {
 
     opts.optopt("d", "", "The path to a database to open", "DATABASE");
     opts.optflag("h", "help", "Print this help message and exit");
+    opts.optopt("q", "query", "Execute a query on startup. Queries are executed after any transacts.", "QUERY");
```

Comment:
Yeah, I was kinda lazy about figuring out how to read the `getopts` flags in sequence. I should probably not be lazy.